Gemfile.lock

```diff
@@ -65,10 +65,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 4f2d386700245582301ae2b04ddd629e422697a0
-  branch: master
+  revision: 17c2992bce1572a67490d76193e0568177810399
+  branch: 10-10cg-new-version
```

Comment:
I don't think it's ok to run vets-json-schema on non-master.

Reply:
This was a mistake, fix here: https://github.com/department-of-veterans-affairs/vets-api/pull/5371